lib/datasource/cdnjs/common.ts

```diff
@@ -0,0 +1,29 @@
+import { z } from 'zod';
+import type { CdnjsResponse } from './types';
+
+/**
+ * See: https://cdnjs.com/api
+ */
+export const Response = z.object({
+  homepage: z.string().url().optional(),
+  repository: z
+    .object({
+      url: z.string().url().optional(),
+    })
+    .optional()
+    .nullable(),
```

Comment:
What does this optional+nullable combination mean compared to just optional? Without `nullable` would it mean that `"repository": null` is treated as a validation error?

Reply:
Optional means, the property may exist (maybe also allows setting to `undefined`).
Null mean allow explicit `null` values.

Reply:
So "yes" to my example?

Reply:
yes, without `nullable` it will throw